### ISSUE TITLE
It doesn't install the .dll surge file

### ISSUE THREAD

**Issue by beitenh** [kurasu/surge#5]
Thank you for this good synth! It is not copyed nor installed any vst file. I don't find it, in any site of the disk. Please your help.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. windows 10]
 - Host [e.g. bitwig, logic]
 - Version [e.g. 2.4]

**Additional context**

**Comment by sagamusix** [comment]
Indeed, the installer provided on the Releases page seems to install about 50MB of files to `%localappdata%\Surge` and the rest (uninstaller) ends up in the path the user actually specified. The DLL is found in neither of those two paths.

Slightly related: The default path the installer wants to install to is `%localappdata\Surge;` including the trailing Semicolon. This seems to not be intentional and maybe has something to do with this issue.

**Comment by Gargaj** [comment]
For me the DLL ended up in ```C:\Program Files\Common Files\VST2\```

**Comment by Adrienlq** [comment]
Hello Hello :) Had the same problem, no prompting about where I need to install the files, found the vst3 and the dll in the directory mentionned above...

**Comment by baconpaul** [comment]
In the upcoming 18 release it will tell you where the VST3 is installed. The answer is:

Q:  Where is Surge synthesizer installed on Windows?
A: Surge is installed to the default VST3 plugin path, which means:

32-bit path - C:\Program Files (x86)\Common Files\VST3\Surge Synth Team
64-bit path - C:\Program Files\Common Files\VST3\Surge Synth Team

We advise you to NOT move the plugin to any other folder on your system. We also advise you not to move the C:\ProgramData\Surge\ folder anywhere else, or bad things will happen!
